pkg/vetter/invalidserviceforjwtpolicy/README-invalid-service-for-jwt-authentication-policy.md

```diff
@@ -0,0 +1,16 @@
+# Invalid Service For JWT Authentication Policy
+
+## Example
+
+The JWT Authentication Policy `your-jwt-authpolicy` in namespace `your-apps` targets service `your-service` in the same
+namespace. The target services port names are not prefixed with mesh supported protocols.
+
+## Description
+
+## Suggested Resolution
```

Comment:
Can we add examples of AuthPolicies and corresponding service with incorrect port names? Makes it easier to understand.

You can use the same example in the other vetter and explain what service users should create.